resources/event-bus/charts/natss-knative-provisioner/values.yaml

```diff
@@ -0,0 +1,12 @@
+# Default values for natss-knative-provisioner.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+controller:
+  image:
+    pullPolicy: IfNotPresent
+    location: eu.gcr.io/kyma-project/event-bus/knative/natss/controller@sha256:c1561149ac50fa46a210bbdc135d4f2bc9ff427b9bfb25baa83f4562dde50987
+dispatcher:
+  image:
+    pullPolicy: IfNotPresent
+    location: eu.gcr.io/kyma-project/event-bus/knative/natss/dispatcher@sha256:f18b4c53be9e34cbf7ac946738e08722b30e073575b77dbec1fd8f430c0abd6d
```

Comment:
You can use the tag `604de5d6` here instead of the sha.

Reply:
Done